R/markdown-link.R

```diff
@@ -11,18 +11,27 @@
 #' ```
 #' MARKDOWN           LINK TEXT  CODE RD
 #' --------           ---------  ---- --
-#' [fun()]            fun()       T   \\link[=fun]{fun()}
-#' [obj]              obj         F   \\link{obj}
+#' [fun()]            fun()       T   \\link[=fun]{fun()} or
+#'                                    \\link[pkg:file]{pkg::fun()}
+#' [obj]              obj         F   \\link{obj} or
+#'                                    \\link[pkg:obj]{pkg::obj}
```

Comment:
Not sure about this, but consistent with `[pkg::obj]` below:

```suggestion
#'                                    \\link[pkg:file]{pkg::obj}
```